src/utils/components/AddBootableVolumeModal/utils/utils.ts

```diff
@@ -72,6 +83,12 @@ export const createBootableVolume: createBootableVolumeType =
         draftBootableVolume.spec.storage.storageClassName = storageClassName;
       }
 
+      // accessModes and volumeMode are applied by the backend when applyStorageProfileSettings is true
+      if (!applyStorageProfileSettings) {
```

Comment:
i think this comment can be removed as 
draftBootableVolume.spec.storage.accessModes = claimPropertySets?.[0]?.accessModes;
is clear, claimPropertySets is from BE, up to u

Reply:
Done!